packages/remix-deno/index.ts

```diff
@@ -2,6 +2,7 @@ import "./globals.ts";
 export { createFileSessionStorage } from "./sessions/fileStorage.ts";
 export {
   createRequestHandler,
+  createRemixRequestHandler,
```

Comment:
Sounds good. I had though of that initially but we use import it like that in  `packages/remix-deno/server.ts` so I was trying to avoid aliasing it twice. I'll go ahead and make the change.